src/redux/superadmins/actions.js

```diff
@@ -0,0 +1,19 @@
+import {
```

Comment:
You can import all by using `import * as name from constants` , I recommend this to you because you'll have 3 for every endpoint and this will be very messy